Week3/homework/step2-3.js

```diff
@@ -7,7 +7,9 @@ function repeatStringNumTimesWithFor(str, num) {
 
   // Replace this comment and the next line with your code
   console.log(str, num, result);
-
+  for (let i = 0; i < num; i++) {
+    result += `${str}`;
```

Comment:
No need to use a template literal here, this would work just the same because `str` is already a string;
`result += str;`